packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -96,25 +93,23 @@ const Info = ({ handleNavigationClick, handleSelect, initial_index, item, list }
                     />
                 </div>
             )}
-            <Carousel
+            <div
```

Comment:
@henry-deriv why do we make this change in this PR? The scope of this PR covers only tests and TS migration, right? :) You removed carousel in another PR as far as I remember

Reply:
Yes. I merged kate's PR that came through master and fixed types in PR where we removed carousel.

I didn't want to fix types specific to this PR where it's going to be overwritten again anyways. So instead I just kept it all consistent across both PRs since carousel removal is branched off of this PR.

Reply:
I am pretty sure this carousel removal PR https://github.com/binary-com/deriv-app/pull/9352/files is identical to this tech debt PR.

Reply:
@maryia-deriv

Reply:
let's pull the latest from master into this PR branch to see all the differences if there are any :)